.github/workflows/ci.yml

```
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'skipci')"
```

```suggestion
    if: "!contains(github.event.head_commit.message, 'norelease')"
```
i think, to match what's in the version script

---

`npm-verison.sh` already checks for this. I was thinking we could have it still run with `norelease` to verify the build, but perhaps we don't need this with the CI running on each commit to a PR

---

Oops, actually because this is `ci.yml`, this won't apply. The CI should always run unless `skipci` is passed.